client/policy_validator.go

```diff
@@ -300,6 +333,19 @@ type PolicyAPIResult struct {
 	Error     string   `json:"error"`
 }
 
+type allowedBy map[string][]string
+
+func (a allowedBy) Add(key, policy string) {
+	a[key] = append(a[key], policy)
+}
+
+type PolicyDump struct {
```

Comment:
Minor personal nit pick, not crucial, PolicyInfo seems nicer and more informative than PolicyDump.

Reply:
:+1: